kolibri/content/management/commands/content.py

```diff
@@ -0,0 +1,156 @@
+
+        if not os.path.exists(content_directory):
+            self.stderr.write(
+                self.style.ERROR(
+                    '\nCurrent content directory {} does not exist.'
+                ).format(content_directory))
+            raise SystemExit()
+
+        # If the current directory is the same as the destination
+        if content_directory == destination:
+            self.stderr.write(
+                self.style.ERROR(
+                    '\nDestination {} is the current content directory.'
+                ).format(destination))
+            raise SystemExit()
+
+        # Check if Kolibri is running
+        try:
+            server.get_status()
+            self.stderr.write(self.style.ERROR(
+                "Cannot migrate content while Kolibri is running, please run:\n"
+                "\n"
+                "    kolibri stop\n"
+            ))
+            raise SystemExit()
+        except server.NotRunning:
+            pass
+
+        self.migrate(content_directory, destination)
+
+    def migrate(self, src, dst):
+        """
+        Migrate the content from current content directory to the destination.
+        """
+        logger.info('Current content directory is {}'.format(src))
+        logger.info('Migrating the content into {}'.format(dst))
+
+        databases_src = os.path.join(src, 'databases')
+        databases_dst = os.path.join(dst, 'databases')
+        storage_src = os.path.join(src, 'storage')
+        storage_dst = os.path.join(dst, 'storage')
+
+        # Check if destination has content by checking if databases folder is not empty
+        dst_content_exists = os.path.exists(databases_dst) and \
+            os.listdir(databases_dst)
+
+        # If destination has content inside, ask users if they want to keep the content
+        if dst_content_exists:
+            self.handle_user_input(src, dst)
+
+        # copy the databases folder
+        self.copy_content(databases_src, databases_dst)
+        shutil.rmtree(databases_src)
+
+        # copy the storage folder only when the source folder contains it
+        if os.path.exists(storage_src):
+            self.copy_content(storage_src, storage_dst)
+            shutil.rmtree(storage_src)
+
+        self.update_config_content_directory(dst)
+
+    def handle_user_input(self, src, dst):
+        """
+        If destination has content inside, ask users if they want to keep the
+        content in the destination. We will copy the content to destination
+        depending on the user response.
+        """
+        user_answer = input(
+            self.style.WARNING(
+                'The destination {} has content inside.\n'
+                'Do you want to keep the content? (Yes/No)'.format(dst)))
+
+        # If the user does not want to keep the content in the destination,
+        # remove the databases folder and storage folder in the destination
+        # directory
+        if user_answer == 'no' or user_answer == 'No':
```

Comment:
Easier to write: `user_answer.strip().lower() == "no"`, which also works with `  no  `, `NO` and the even more unlikely `nO` :)

Reply:
Actually, this could also be: `user_answer.strip().lower() in ["no", "n"]`